app/code/Magento/CatalogGraphQl/Model/Resolver/Product/PriceRange.php

```diff
@@ -60,7 +60,6 @@ public function resolve(
 
         /** @var Product $product */
         $product = $value['model'];
-        $product->unsetData('minimal_price');
```

Comment:
I am sure this had some other purpose and we probably broke something else here...